lib/stream-utils.ts

```diff
@@ -33,10 +34,22 @@ export async function streamToBuffer(stream: Readable): Promise<Buffer> {
   });
 }
 
-export async function streamToHash(stream: Readable): Promise<string> {
+export async function streamToHashSHA1(stream: Readable): Promise<string> {
+  return doStreamToHash(stream, HASH_ALGORITHM_SHA1, HASH_ENCODING);
+}
+
+export async function streamToHashSHA256(stream: Readable): Promise<string> {
+  return doStreamToHash(stream, HASH_ALGORITHM_SHA256, HASH_ENCODING);
+}
+
+export async function doStreamToHash(
+  stream: Readable,
+  hashAlgo: string,
+  hashEnc: string,
```

Comment:
IMO it's better to be explicit rather than shorten var names: `hashAlgorithm` and `hashEncoding`; it's just better for readability

Reply:
Agreed .fixed